### PR TITLE
#60 - Moddified registration controller to support username and password

### PR DESCRIPTION
**Pull request by adnan-saab**
Moddified registration controller to support username and password and modified to file name to comply with appropriate syntax.

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ db.on('error', function(err){
 
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
+var usersRouter = require('./routes/usersController');
 var picsRouter = require('./routes/picsController');
 var loginRouter = require('./routes/login');
-var users = require('./routes/users');
+var users = require('./routes/usersController');
```
Comment (BozhidarLeshev) [inline api/app.js:30]: why two routers

Comment (adnan-saab) [reply api/app.js:30]: the one in red means it will get deleted so I will only have on router

Comment (BozhidarLeshev) [reply api/app.js:30]: But you have two: one users and one usersRouter

File: api/app.js [hunk 2]
```diff
@@ -23,10 +23,10 @@ db.on('error', function(err){
 
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
+var usersRouter = require('./routes/usersController');
```
Comment (BozhidarLeshev) [inline api/app.js:26]: Here is the second

File: api/app.js (hunk 1 above)
Comment (adnan-saab) [reply api/app.js:30]: Oh I see, I will delete one.

**Pull request merged**